SPECS/grep/grep.spec

```diff
@@ -47,9 +60,13 @@ make  %{?_smp_mflags} check
 %defattr(-,root,root)
 
 %changelog
+* Tue Jun 15 2021 Andrew Phelps <anphel@microsoft.com> 3.1-5
+- Support perl regular expressions ("grep -P")
```

Comment:
I've been waiting so long for this, thank you. :)

Reply:
no problem!